src/Shared/Converters/BondStatisticsConverter.ts

```diff
@@ -0,0 +1,127 @@
+/* *
+ *
+ *  (c) 2009-2025 Highsoft AS
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ *  Authors:
+ *  - Askel Eirik Johansson
+ *
+ * */
+
+
+'use strict';
+
+
+/* *
+ *
+ *  Imports
+ *
+ * */
+
+
+import {
+    SecurityDetailsConverterOptions,
+    SecurityDetailsMetadata
+} from '../../SecurityDetails/SecurityDetailsOptions';
+import SecurityDetailsJSON from '../../SecurityDetails/SecurityDetailsJSON';
+import MorningstarConverter from '../MorningstarConverter';
+
+/* *
+ *
+ *  Class
+ *
+ * */
+
+
+export class BondStatisticsConverter extends MorningstarConverter {
+
+
+    /* *
+     *
+     *  Constructor
+     *
+     * */
+
+
+    public constructor (
+        options?: SecurityDetailsConverterOptions
+    ) {
+        super(options);
+
+        this.metadata = {
+            columns: {},
+            ...(options && options.hasMultiple && { ids: [] }),
```

Comment:
Change to optional chaining in all occurrences of that

```suggestion
            ...(options?.hasMultiple && { ids: [] }),
```